src/main/java/com/leverx/springapp/demo/controller/UserController.java

```diff
@@ -0,0 +1,52 @@
+package com.leverx.springapp.demo.controller;
+
+import com.leverx.springapp.demo.model.User;
+import com.leverx.springapp.demo.repository.UserRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import javax.validation.Valid;
+import java.net.URI;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+    private UserRepository userRepository;
+
+    public UserController(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    @PostMapping
+    public ResponseEntity createUser(@Valid @RequestBody User user) {
+        userRepository.save(user);
+        URI location = ServletUriComponentsBuilder
+                .fromCurrentRequest().path("/{id}")
+                .buildAndExpand(user.getId()).toUri();
+        return ResponseEntity.created(location).build();
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity get(@PathVariable("id") Long id) {
+        if (userRepository.findById(id).isPresent()) {
+            return new ResponseEntity<Object>(userRepository.findById(id), HttpStatus.OK);
```

Comment:
@blackavgn   ResponseEntity< Object > 🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔

Reply:
@blackavgn Please refresh your knowledge regarding generics, We will **discuss** them.

Reply:
I used it because of checkstyle.It doesn't identify <> empty braces.
Non sense for me too.

Reply:
@blackavgn in this case you should to check checkstyle rules and understand why it fails build. What checkstyle rule was triggered? Maybe checkstyle mix up rules. Please get all information and then ask question - such workarounds not allowed.